src/test/mockFactory.ts

```diff
@@ -300,6 +312,12 @@ export const createTestHldLifecyclePipelineYaml = (
       vmImage: VM_IMAGE
     },
     steps: [
+      {
+        task: "HelmInstaller@1",
+        inputs: {
+          helmVersionToInstall: "2.16.3"
```

Comment:
@mtarng Can we make this the helm version a constant?

Reply:
Yes, I'll follow up on that.